HungarianAlgorithm/Application/Program.cs

```diff
@@ -1,3 +1,22 @@
 ﻿
```

Comment:
Nie chce mi się kompilować, brakuje chyba tej linijki
```suggestion
﻿using Hungarian;
```
Poza tym - profeska :)

Reply:
Tak, using się zgubił :)